qaoa_vrp/build_circuit.py

```diff
@@ -165,7 +167,7 @@ def qubo_to_qaoa(qubo, num_layers):
     op, offset = qubo.to_ising()
 
     quantum_instance = QuantumInstance(
-        BasicAer.get_backend("statevector_simulator"),
+        QasmSimulator(method='matrix_product_state'),
```

Comment:
Is this used? If not get rid of it.

Otherwise can you pass through `matrix_product_state` as an argument